lib/deserialize.js

```diff
@@ -49,6 +49,25 @@ module.exports = function (app) {
       //TODO: Rewrite to normal search model by type and FK
     }
 
+		/**
+		 * Handle include relationship requests (aka sideloading)
+		 */
+		if (RelUtils.isRequestingIncludes(ctx.req)) {
+      ctx.res.set({'Content-Type': 'application/vnd.api+json'});
+
+			if (!RelUtils.shouldIncludeRelationships(ctx.req.method)) {
+				return next(RelUtils.getInvalidIncludesError());
+			}
+		
+			var include = RelUtils.getIncludesArray(ctx.req.query);
+			include = include.length === 1 ? include[0] : include;
+
+			ctx.req.remotingContext.args = ctx.req.remotingContext.args || {};
+			ctx.req.remotingContext.args.filter = ctx.req.remotingContext.args.filter || {};
+			ctx.req.remotingContext.args.filter.include = include;
```

Comment:
I have a sneaking suspicion that ctx.req.remotingContext is a circular reference to ctx. Will follow up after looking through the rest of the code